tests/unit/test_response.txt

```diff
@@ -38,16 +54,10 @@ Test group_granule_ids:
     ...             "failed": 123,
     ...             "report": {
     ...                 "HLS.L30.ABC.v2.0.B01.tif": {
-    ...                     "granuleId": "HLS.L30.ABC.v2.0",
-    ...                     "status": "failed"
     ...                 },
```

Comment:
(genuine question, I'm not sure what the report looks like)

Is this up to date? It seems a little odd to me to have an empty dict as the "value" to each filename "key"

Reply:
Makes sense! 🚀